processing-and-analysis/02-command-files/Ganling's file/Random_effect_Interaction.Rmd

```diff
@@ -132,10 +127,35 @@ AIC(mod7, mod8)
 ```
 #### NOTE: The AIC differences are all greater than 2 when interacting with different demographics.  
 
+### Models without unique `ta_sect`
+```{r}
+df5_true <- rename(data_2s_exam_score_true)
 
+#First, test that the treatment impacts scores
+mod9 <- lm(finalexam ~ experiment1 + satmath + satverbal + mastered_topics_initial_kc + high_sch_gpa, 
+           data=df5_true)
+summary(mod9)
+AIC(mod9)
 
+#Include random effects (clustering)
+mod10 <- lmer(finalexam ~ experiment1 + satmath + satverbal + mastered_topics_initial_kc + high_sch_gpa + (1|ta_sect), 
+           data=df5_true)
+summary(mod10)
+AIC(mod10)
 
-
+```
+```{r}
+AIC(mod9,mod10)
+```
+#### Values with unique `ta_sect` are: 	
+df
+<dbl>
+AIC
+<dbl>
+mod1	7	63734.08		
+mod2	8	63561.45
+
+#### NOTE: When considering random effects with unique `ta_sect`, the AIC value decreases compared to the AIC value without unique `ta_sect`
```

Comment:
Here, it appears that you are comparing the model without a random effect (mod9) to the model with `ta_sect` as a random effect, both applied to the same dataset which has non-unique values of `ta_sect`. I don't think you can make a conclusion about the relative fit of models applied to data with non-unique `ta_sect` values and to data with unique `ta_sect` values.

To make the comparison I was thinking of, the same model:

`finalexam ~ experiment1 + satmath + satverbal + mastered_topics_initial_kc + high_sch_gpa + (1|ta_sect)`

would be applied to different data sets: one with unique `ta_sect` values, and one without.